addons/website_sale/views/templates.xml

```diff
@@ -1495,7 +1495,7 @@
                         <div class="col-12 col-xl-auto order-xl-2">
                             <t t-call="website_sale.cart_summary"/>
                         </div>
-                        <div class="col-12 col-xl order-xl-1 oe_cart">
+                        <div class="col-12 col-xl order-xl-1 oe_cart mb32">
```

Comment:
here I added the margin on the col because we dont know what will be the last elements